docs/prometheus-integration.md

```diff
@@ -116,3 +116,165 @@ The configuration file above can be used to deploy Prometheus Server with
 scraping configuration for Antrea services.
 To deploy this configuration use
 `kubectl apply -f build/yamls/antrea-prometheus.yml`
+
+# Antrea Prometheus Metrics
+Antrea Controller and Agents expose various metrics, some of which are provided 
+by the Antrea components and others which are provided by 3rd party components 
+used by the Antrea components.
+
+Below is a list of metrics, provided by the components and by 3rd parties.
+
+## Antrea Agent Metrics
+**antrea_agent_conntrack_antrea_connection_count:** Number of connections
```

Comment:
formatting-wise, I think that markdown tables would be nice, but it's pretty subjective and can be postponed to a future PR

Reply:
@antoninbas I agree. However some metric names are very long, meaning that this will exceed 80 chars or Just look very awkward.
Look at antrea_controller_applied_to_group_sync_duration_milliseconds for example - it looks odd enough even without a table.

Reply:
let's figure it out later, after we see what it looks like in the website